app/views/classifications/_document_list_from_rummager.html.erb

```diff
@@ -0,0 +1,34 @@
+<%
```

Comment:
It seems that this is trying to be the same as `document_list.html.erb` but with specific adaptions because Rummager results are slightly different(?) If so and you have the time, I'd suggest trying to find a way to make it so `document_list.html.erb` can present both - perhaps by either having a class that will take in the Rummager responses and respond in the same way as the original responses or by having some class that can take in either kind of response and act like a common interface for them. Otherwise, someone making a change to the document list will have to figure out that they are meant to be the same and make the same changes twice. This might also help with the problem we discussed about FeedHelper (app/helpers/feed_helper.rb line 9)
If they're different files because they fundamentally are meant to be different, ignore all of this!

Reply:
That's a valid question. Initially I had planned to update the `document_list` template so that it could take both Rummager and Whitehall documents.

Since the aim is to migrate document lists from Whitehall to Rummager, I thought it would be easier to manage two similar but separate templates rather than try to making one adjust to two models. Having this separation of concerns is intended to keep the migration and views simpler, and once we're done it'll let us just delete the Whitehall template rather than needing to untangle it from a combined view. You're right that having to manage two views that essentially have the same task is a bit odd, even if the models they're provided with are different. Let me know if having two views isn't simpler - I'll be happy to have another go at a common view that plays nice with both models.

Reply:
Ah, right. That does make sense. In which case, I'd suggest making this super clear somewhere <waves hands indicating I'm not really sure where, maybe the PR notes, or a commit message> that this is the plan so other devs will know where you're coming from

Reply:
Yeah that's a good idea, sorry this wasn't clear! I will update the commit message.